### DISCUSSION TITLE
Counter-arguments to the original article by Salem against RMS that started it all

### DISCUSSION THREAD

**Discussion by ghost** [rms-support-letter/rms-support-letter.github.io#4083] [General]
### Found on responses to her medium article

<details>
<summary> " you are doing is an ad hominem attack " </summary>
What you are doing is an ad hominem attack that also implies personal sanctions. That is against the rules of civic discourse.
Marvin Minsky is dead. He cannot defend himself against accusations. He cannot clear his name in court. But Richard criticises the language of the accusers:

>  “The word “assaulting” presumes that he applied force or violence, in
some unspecified way, but the article itself says no such thing.
Only that they had sex.”

Richard Stallman, a autist with whom his closest peers happily disagree and have their stories of hosting him, just does what Richard does. He rejects imprecise language and he objects to terms that distort the mind.

It is legally wrong to use the term assault. And it is even not established (yet possible) that the girl Minsky allegedly had sex with was underage at that time. The age of consent differs internationally, in my jurisdiction it is 16. Of course that does not imply underage prostitution as in the Epstein billionaire case. Having sex with 16 year olds is not acceptable to me (nor is sex with 23 year olds) but also not “child rape”/pedophilia — in my understanding as a child would be a minor below 14.

From this originates a social media slander machine of shooting the messenger. RMS never was accused of being pedophile himself, he does not attract women either. But quoting your article Sarah Mei sent off the following tweet (and myriads other personal attacks):
> I’ve tweeted for many years about how awful Richard “RMS” Stallman is — the pedophilia, the ableism, the misogyny.

followed by a straw man argument:
Inevitably, each time I do, dudes examine my receipts & then say “all those incidents are from years ago! he’s changed now!”
I think no one claimed that he changed. He revised a former argument that was criticised. But I think that her’s was a straw man argument to start with. There is no evidence that Richard ever engaged in acts of pedophilia himself but indications that he hold controversial libertarian opinions on that matter in the past that he revised more recently.

And then you have the aggressive and mean campaign to strip Richard from his FSF board role and turn him into a wandering priest — which is a pretty funny demand to anyone who knows the details.

Generally: RMS was factually right to defend Minsky against distorted language and if you disagree with him you need to argue against his nitpicking argument, not make massive outrage and slander and ad hominem attacks, and invite others to share their RMS awkwardness anecdotes etc etc.

“I am upset” is no argument.

Disclaimer: I am very critical of RMS for other unrelated reasons but I find it noble in the age of social media to defend a (deseased) person under attack. So I did the same and defended RMS. We have to fight the Hate Machinery in social media, the slander, the character assasinations, the persons who make us believe that wrong opinions are worse than wrong actions, the intolerance, the use of hate mobs that demand to reprimand persons based on their kangeroo court imaginations.

https://medium.com/@meddlleu/what-you-are-doing-is-an-ad-hominem-attack-that-also-implies-personal-sanctions-e4997a30bf0c
</details>

<details>
<summary> "cancel culture is non-sense" </summary>
Dont be an angry virtuous outrage mob. we understnad you are upset but what is your actual intent here? it seems like it is to tarnish this person’s reputation and damage their entire career and livelihood…simply because he made comments that you find objectionable? and all the virtue signalers here will give you claps because no one will stand up for common sense here and say anything defending the person being attacked. this “cancel culture” stuff is nonsense.it’s quite narcissistic on your end to be honest. you are so morally outraged by comments you find objectionable that you want to wage a public campaign against this person with the sole purpose of damaging his reputation and livelihood. that is shameful to be honest.

https://medium.com/@LABachelor/dont-be-an-angry-virtuous-outrage-mob-8b7deba046ae?source=responses-----fec6ec210794----16----------------------------
</details>

<details>
<summary> "Inappropriate choice of words may not be a good enough reason"  </summary>
Inappropriate choice of words may not be a good enough reason to expel Stallman. Perhaps you should be removed for not writing an official plea but instead publicly denigrading the reputation of MIT.

https://metacritical.medium.com/inappropriate-choice-of-words-may-not-be-a-good-enough-reason-to-expel-stallman-d408d96c1bb0?source=responses-----fec6ec210794----5----------------------------
</details>

<details>
<summary> " He has a rational and logical reasoning to all his “controversial” statements" </summary>

Richard Stallman speaks the truth, and the SJWs pillory him for it. As so many other people said, go email him, go to his office and talk to him. He has a rational and logical reasoning to all his “controversial” statements that he has made over the years. But be warned: he will not kowtow to any virtue signalling, moral-posturing or bad arguments hidden behind appealing to societial prejudices. I’m glad we have redpilled individuals such as RMS who absolutely do not give a shit about the popular prejudices of their society, and I hope he continues to speaks his opinons no matter who it offends. Especially given RMS isn’t exactly known for bullying people, cursing people out, being mean to people, etc. He will just make his political positions clear. And for disclamer, yes I am supportive of his Free Software positions and philosophy even if I wouldn’t personally only use free software (ie not using netflix, a smart phone, google, etc)

https://medium.com/@warsie/richard-stallman-speaks-the-truth-and-the-sjws-pillory-him-for-it-d96ecd7c4217?source=responses-----fec6ec210794----10----------------------------
</details>

<details>
<summary> "SJWs want to get rid of all men in STEM with fake accusations of rape and sexual assault"  </summary>

To Kill a Mockingbird is being banned because an African-American man is framed with rape and sexual assault he didn’t do. SJWs want to get rid of all men in STEM with fake accusations of rape and sexual assault. This is what RMS is addressing.

https://medium.com/@richardsmith_76348/to-kill-a-mockingbird-is-being-banned-because-an-african-american-man-is-framed-with-rape-and-1b4e2b3f7440?source=responses-----fec6ec210794----14----------------------------
</details>

**Comment by shenlebantongying** [comment]
If possible, I think creating a new page on GitHub's [Wiki](https://github.com/rms-support-letter/rms-support-letter.github.io/wiki) might help to preserve the content.

**Comment by nukeop** [comment]
Can you please move this to the wiki or discussions?

**Comment by nukeop** [comment]
And don't use the cringe 'sjw' term which means nothing
